<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncrytpDecryptSymmetric</class>
 <widget class="QDialog" name="EncrytpDecryptSymmetric">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>235</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Encrypt symetric</string>
  </property>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="old_file">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>old file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_key">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>chose File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_file">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>new file</string>
   </property>
  </widget>
  <widget class="QLabel" name="old_file_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="new_file_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressCrypt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::BottomToTop</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="WriteKey">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>write password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="or_lbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Or</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RemoveAfter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>remove after</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
